src/main/play-doc/operation/Install.md

```diff
@@ -721,6 +721,10 @@ Wait until the ConductR instance's health to marked as healthy before proceeding
 
 If more instances are required, scale the application to the desired total and ensure that all of the instances are healthy before proceeding.
 
+### Executor Port Reservation
+
+Upon startup ConductR will launch its agent process on each of the Mesos slave nodes. ConductR Agent process will reserve port `2552` for Akka remoting and port range `10000-10999` for exposing endpoints of the bundle deployed within ConductR.
+
```

Comment:
I think we do since this document is valid for our released beta version. If only `10000-10999` is mentioned, then the document will be inaccurate.

We should remove the mention of `2552` as part of the cluster client replacement work, so the document will always be correct - wdyt?